<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MelodiMix</class>
 <widget class="QMainWindow" name="MelodiMix">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MelodiMix</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: &quot;black&quot;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="Nav_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>221</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#Nav_frame {

	border-right:2px solid gray;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="home_nav">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-bottom: 2px solid #C1C1C1;
color:&quot;white&quot;;
background-color:&quot;black&quot;;</string>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="logo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="home_icon">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>18</width>
       <height>18</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:&quot;black&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources.qrc">:/img/img/home.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="fav_nav">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-bottom: 2px solid #C1C1C1;
color:&quot;white&quot;;</string>
     </property>
     <property name="text">
      <string>Favourites</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="fav_icon">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>18</width>
       <height>18</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:&quot;black&quot;;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources.qrc">:/img/img/fav.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="import_nav">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-bottom: 2px solid #C1C1C1;
color:&quot;white&quot;;</string>
     </property>
     <property name="text">
      <string>Import</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="import_icon">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>310</y>
       <width>18</width>
       <height>18</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:&quot;black&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources.qrc">:/img/img/import.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="search_nav">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>260</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-bottom: 2px solid #C1C1C1;
color:&quot;white&quot;;</string>
     </property>
     <property name="text">
      <string>Library</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="search_icon">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>18</width>
       <height>18</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:&quot;black&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources.qrc">:/img/img/search.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="Pages">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>581</width>
      <height>591</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="HomePage">
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>571</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>361</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="accessibleDescription">
        <string>Enter song name</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:&quot;white&quot;;
color:&quot;black&quot;;
border-radius:&quot;15px&quot;;
padding:&quot;10px&quot;</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>20</y>
         <width>81</width>
         <height>35</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>35</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:&quot;white&quot;;
color:&quot;black&quot;;
border-radius:&quot;15px&quot;</string>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </widget>
     <widget class="QListWidget" name="music_list">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>521</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QFrame" name="player_frame">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>500</y>
        <width>581</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#player_frame{

background-color:&quot;black&quot;;
border:1px solid white;

}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="player_icon">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;
border-radius:40;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="Resources.qrc">:/new/prefix1/img/disk.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="song_title">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QSlider" name="progressbar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>571</width>
         <height>10</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>10</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>10</height>
        </size>
       </property>
       <property name="font">
        <font>
         <kerning>true</kerning>
         <hintingpreference>PreferDefaultHinting</hintingpreference>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="tabletTracking">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: &quot;green&quot;;
border:&quot;none&quot;;</string>
       </property>
       <property name="pageStep">
        <number>0</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
       <property name="tickInterval">
        <number>0</number>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="SearchPage">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:&quot;red&quot;;</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="FavPage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:&quot;white&quot;;
</string>
      </property>
      <property name="text">
       <string>Favorite</string>
      </property>
     </widget>
     <widget class="QListWidget" name="fav_list">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>541</width>
        <height>491</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ImportPage">
     <widget class="QLabel" name="import_page_header">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:&quot;white&quot;</string>
      </property>
      <property name="text">
       <string>Import song files</string>
      </property>
     </widget>
     <widget class="QPushButton" name="import_btn">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:&quot;black&quot;;
background-color:&quot;white&quot;;
border-radius:&quot;20px&quot;</string>
      </property>
      <property name="text">
       <string>+ Import </string>
      </property>
     </widget>
     <widget class="QListWidget" name="imported_songs_list">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>231</y>
        <width>511</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="imported_files_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>431</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:&quot;white&quot;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
